### PR TITLE
Refs #6341 Adapted documentation to the Fast-RTPS-Gen emancipation. [6353]

### PR DESCRIPTION
**Pull request by LuisGP**
Fast-RTPS-Gen is not going to be directly available within Fast-RTPS and will need a separate build process.

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/47/)

File: docs/geninfo.rst [hunk 1]
```diff
@@ -31,6 +31,20 @@ It also provides an initial implementation of a publisher and a subscriber using
 Compile
 -------
 
-In order to compile *fastrtpsgen* you first need to have `gradle <https://gradle.org/install>`_ and `java JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_ installed (please, check the JDK recommended version for the gradle version you have installed).
+In order to compile *fastrtpsgen* you first need to have `gradle <https://gradle.org/install>`_ and
+`java JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_ installed
+(please, check the JDK recommended version for the gradle version you have installed).
 
-To generate *fastrtpsgen* you will need to add the argument ``-DBUILD_JAVA=ON`` when calling CMake.
+If you want to compile *fastrtpsgen* java application, you will need to download its source code from
```
Comment (richiware) [inline docs/geninfo.rst]: ¿"If you want"? If the user is reading this second paragraph, sure he want to compile it :). I think you can remove those words.

Comment (LuisGP) [reply docs/geninfo.rst]:
```suggestion
To compile *fastrtpsgen* java application, you will need to download its source code from
```

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/49/)

**Pull request merged**